src/LightSm.csx

```diff
@@ -31,50 +31,33 @@ void PrintSmInfo(StateMachine sm)
     BehaviorDescriber describer = new(singleLineFormat: true);
 
     InitialState rootInitialState = sm.ChildType<InitialState>();
-    Console.WriteLine($"Root initial state behaviors:");
-    PrintBehaviors(rootInitialState, describer);
-    Console.WriteLine($"");
-
-    // The below code will visit all states in the state machine.
-    // A more complete solution would also visit pseudo states like initial, choice, enter and exit points.
-    // Let's start simple for now and ignore pseudo states.
-    sm.VisitTypeRecursively((State state) =>
-    {
-        Console.WriteLine($"State {state.Name} behaviors:");
-        // could use name to generate a test function name
-
-        PrintBehaviors(state, describer);
-        Console.WriteLine("");
-    });
-}
-
-// Note that this function takes a vertex, which is a base class for states and pseudo states.
-static void PrintBehaviors(Vertex vertex, BehaviorDescriber describer)
-{
-    foreach (var behavior in vertex.Behaviors)
-    {
-        // To learn how to inspect a behavior for transitions, guards, ...
-        // see https://github.com/StateSmith/StateSmith/blob/41d7b6bb663dac3d99c0223a15dda3638cd21548/src/StateSmith/SmGraph/BehaviorDescriber.cs#L8
-        Console.WriteLine("    " + describer.Describe(behavior));
-    }
-}
-
-
-
-
 
+    // Imports
+    Console.WriteLine($"import {{jest}} from '@jest/globals';");
+    Console.WriteLine($"import {{ {sm.Name} }} from './{sm.Name}.js';");
+    Console.WriteLine($"");
 
+    // beforeEach
+    Console.WriteLine($"beforeEach(() => {{");
+    Console.WriteLine($"    jest.clearAllMocks();" );
+    Console.WriteLine($"}});");
+    Console.WriteLine($"");
 
+    // TODO is it a valid assumption to assume that every state machine has at least two vertices?
+    // TODO is it a valid assumption to assume that the zeroth vertex is the entry vertex, and the 
+    //      next vertex is the next reachable state from the vertex?
```

Comment:
I don't think we can rely on the order of named vertices. It might be dependent on the order in which they appear in the diagram.